containers/CompoundAndEarn/index.js

```diff
@@ -41,41 +41,25 @@ const CompoundAndEarn = () => {
   const { getLastSnowballInfo } = useAPIContext();
   const snowballInfoQuery = getLastSnowballInfo();
   const { userPools, userDeprecatedPools, loadedDeprecated,
-    sortedUserPools,setLoadedDeprecated,setSortedUserPools,
-    setUserPools } = useCompoundAndEarnContract();
+    sortedUserPools,setLoadedDeprecated,setSortedUserPools } = useCompoundAndEarnContract();
 
   const [search, setSearch] = useState('');
   const [type, setType] = useState('apy');
   const [userPool, setPool] = useState('all');
   const [lastSnowballInfo, setLastSnowballInfo] = useState([]);
-  const [lastSnowballModifiedInfo, setLastSnowballModifiedInfo] = useState([]);
-  const [filterDataByProtocol, setFilterDataByProtocol] = useState([]);
-  const [loadedSort, setLoadedSort] = useState(false);
-
-  //reset state when index opened
-  useEffect(()=>{
-    if(!loadedSort){
-      setSortedUserPools(false);
-      setLoadedSort(true);
-    }
-  },[loadedSort,setSortedUserPools])
```

Comment:
We need this, because if you go to another page and get back to C&E page after your pools are loaded, they will not load again.
![image](https://user-images.githubusercontent.com/20801365/133463944-46a5df24-ca5f-45aa-ac7d-941489d23a7f.png)